pkg/k8s/endpoints.go

```diff
@@ -119,10 +120,10 @@ func ParseEndpoints(ep *slim_corev1.Endpoints) (ServiceID, *Endpoints) {
 
 	for _, sub := range ep.Subsets {
 		for _, addr := range sub.Addresses {
-			backend, ok := endpoints.Backends[addr.IP]
+			backend, ok := endpoints.Backends[cmtypes.NewIPCluster(addr.IP, 0)]
```

Comment:
Is everything in k8s/endpoints always going to refer to the local cluster?

Reply:
Yes, ClusterID 0 is treated as a local cluster.